### ISSUE TITLE
"Go Back" command should not switch the focused Group

### ISSUE THREAD

**Issue by ManuelEibl** [microsoft/vscode#140862]
TLDR:
Go Back should be limited to the group one is focused on and, hence, open a file one is going back to in that same group view.

When one opens the same file in two groups next to each other, for example, because one is comparing an implementation further down of the file with one at the beginning of the file, a very annoying "feature" can be seen with regards to the Go Back command.
When one opens a new file in group 1 and then uses the Go Back command, the cursor will jump to the already open file in group 2.

This is very annoying and should be changed IMO.
The reason is, that in such a scenario, the group 1 is the group one is actively working on while group 2 is your lookup reference. 

So for example I'm changing something in the implementation while comparing to the implementation in another location, switch file in group 1 to run a test, want to go back to the same editing view as before by using Go Back and end up with my reference being moved away (as the cursor jumps to the group 2 in a different line I was looking at) and in group 1 I still see the code I wanted to go back from. That's not desirable behavior IMO and it is really annoying in that type of working setup, and I wouldn't know in which scenario one would want that.

**Steps to reproduce:**
1. Open the same file in group 1 and 2 next to each other in VS Code.
2. In group 1 open another file
3. Use the Go Back command to try to go back to the original file

**Behavior:**
Group 2 shows the cursor position you have been at in group 1 before.

**Expected behavior:**
Group 1 shows the same view as before, i.e., the cursor position in the file and line you have had open in group 1 before opening the second file.

Version of VS Code: 1.63.2
OS: WSL2 on Windows 10
